### ISSUE TITLE
Light/dark mode toggle no longer works

### ISSUE THREAD

**Issue by thisismikovic** [stephenou/fruitionsite#289]
It seems that Notion changed the structure of the HTML so the onDark and onLight functions no longer target the right thing. I was playing around a bit in the console and I can pinpoint certain elements such as background, but this isn't a good solution as this setting should target the whole theme (colors of text, links, header, whatever else).

Did anyone figure out a more general approach that would set the _theme_ to be light or dark, instead of separate elements?

**Comment by zer0s000-g** [comment]
Indeed. I just noticed that today. Anyone knows how to make the page to be dark by default? my script did not work anymore
